README.md

```diff
@@ -52,3 +52,12 @@ $paginator = new Paginator();
 
 $paginator->get($records, $limit);
 ```
+
+### Flash messages
+
+Simply fire a flash message usings sessions
+```
+$flash = new TrendingRepos\Core\Session;
+$flash->set('success', 'Some message to display!');
```

Comment:
This way you force clients to remember different types and how they are spelled. A better approach would be to introduce constants like `Flash::TYPE_SUCCESS`, etc. The best approach would be to hide these implementation details behind public methods such as `setSuccess()`, etc.